### ISSUE TITLE
Bug: Webpack plugin crashes when run on Webpack 5

### ISSUE THREAD

**Issue by JustinPierce** [ngneat/transloco-keys-manager#98]
### Current behavior
The Webpack plugin fails on Webpack 5 due to a change in a private API.

### Expected behavior
The Webpack plugin should not fail in Webpack 5. ;)

### Steps To Reproduce
Create a basic Angular 12 project, which uses Webpack 5 by default, and follow the basic directions for installing Transloco and the Keys Manager:
1. `ng new test-app`
2. `ng add @ngneat/transloco` (accept defaults)
3. `ng g @ngneat/transloco:keys-manager` (I chose "both" strategy, but I assume just "Webpack" will do)
4. `npm start`

The Angular server will start up and then fail with the following error:

```
✖ ｢wdm｣: TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at /Users/me/src/test-app/node_modules/@ngneat/transloco-keys-manager/webpack-plugin.js:1:985
    at Hook.eval [as callAsync] (eval at create (/Users/me/src/test-app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:12:1)
    at run (/Users/me/src/test-app/node_modules/webpack/lib/Watching.js:142:33)
    at /Users/me/src/test-app/node_modules/webpack/lib/Watching.js:129:6
    at /Users/me/src/test-app/node_modules/webpack/lib/HookWebpackError.js:68:3
    at Hook.eval [as callAsync] (eval at create (/Users/me/src/test-app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
    at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (/Users/me/src/test-app/node_modules/tapable/lib/Hook.js:18:14)
    at Cache.endIdle (/Users/me/src/test-app/node_modules/webpack/lib/Cache.js:140:22)
    at run (/Users/me/src/test-app/node_modules/webpack/lib/Watching.js:126:32)
```

### Keys manager config
```
module.exports = {
  rootTranslationsPath: 'src/assets/i18n/',
  langs: ['en', 'es'],
  keysManager: {}
};
```

### Environment
<pre><code>- Keys manager version: 2.7.2 
- Node version: v12.20.0
- Platform:  Mac
</code></pre>

### Contribution
I would like to make a pull request for this feature:
[👍🏻] Yes! 🚀
[ ] Maybe next time

**Comment by j0nm1** [comment]
Hi, I'm encountering the same issue when updating to Angular 12. Is there any chance that this gets fixed/merged anytime soon?

**Comment by JustinPierce** [comment]
> Hi, I'm encountering the same issue when updating to Angular 12. Is there any chance that this gets fixed/merged anytime soon?

As a workaround, you can copy the Webpack plugin source from this project into your own project and apply the fix to it (it's a one-line fix). Then, call your copy of the plugin directly from your Webpack configuration instead of the one bundled with the keys manager.

**Comment by JustinPierce** [comment]
> Hi, I'm encountering the same issue when updating to Angular 12. Is there any chance that this gets fixed/merged anytime soon?

@j0nm1 As a workaround, you can copy the Webpack plugin source from this project into your own project and apply the fix to it (it's a one-line fix). Then, call your copy of the plugin directly from your Webpack configuration instead of the one bundled with the keys manager.

**Comment by shaharkazaz** [comment]
@JustinPierce can you open a PR? 😄

**Comment by shaharkazaz** [comment]
@JustinPierce @j0nm1  Released in v2.7.3, can you please verify? 🙂

**Comment by JustinPierce** [comment]
> @JustinPierce @j0nm1 Released in v2.7.3, can you please verify? 🙂

@shaharkazaz I'm up and running on 2.7.3 and Webpack 5. Looking good!

**Comment by j0nm1** [comment]
I tested it as well and it works great. Thank you for the fast solution @shaharkazaz and @JustinPierce. 🎉
